reagurk/src/components/footer.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+
+export default function Footer(): JSX.Element {
+
+  return (
+    <>
+      <footer className="text-gray-400 bg-gray-800 body-font fixed
+             inset-x-0
+             bottom-0">
+        <div className="container px-5 py-8 mx-auto flex items-center sm:flex-row flex-col">
+          <a className="flex title-font font-medium items-center md:justify-start justify-center text-white">
+            <img src="https://cdn.discordapp.com/emojis/792081164466126868.png?size=56" alt="gurkult logo" />
+            <span className="ml-3 text-xl">Gurkult</span>
+          </a>
+          <p className="text-sm text-gray-400 sm:ml-4 sm:pl-4 sm:border-l-2 sm:border-gray-800 sm:py-2 sm:mt-0 mt-4">© 2021 Gurkult 
+           
+          </p>
+          <span className="inline-flex sm:ml-auto sm:mt-0 mt-4 justify-center sm:justify-start">
+            <a className="text-gray-400"
+              href='https://github.com/gurkult'>
+              <img src="https://img.icons8.com/ios-glyphs/50/ffffff/github.png" alt="github logo" />
+            </a>
+            <a className="ml-3 text-gray-400"
+              href='https://discord.gg/W9DSfryp8Y'>

```

Comment:
Sorry, missed this one. This should reference the constant we have defined for the server invite.

Reply:
Ah